src/NumericalAlgorithms/LinearOperators/PartialDerivatives.cpp

```diff
@@ -121,45 +124,150 @@ auto logical_partial_derivative(
   return result;
 }
 
+template <typename SymmList, typename IndexList, size_t Dim,
+          typename DerivativeFrame>
+void partial_derivative(
+    gsl::not_null<TensorMetafunctions::prepend_spatial_index<
```

Comment:
(nitpick) make these `gsl::not_null` arguments `const` (the pointer doesn't change, only the data it is pointing to)